.mailmap

```diff
@@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
 Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
+Song Baisen <song.baisen@zte.com.cn>
```

Comment:
Well, that's for normalisation : original entry is like that : songbaisen <song.baisen@zte.com.cn>
I assumed Song Baisen is better here.